test/built-ins/BigInt/asIntN/arithmetic.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
```

Comment:
I guess yes, so `sec-bigint.asintn`. If it's still cloudy, let's keep `esid: pending`. I just need to include it into my future tasks as soon as it is defined.